pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "django-public-admin"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
Can you revert this change and rebase?

I’m really sorry it’s not documented, but I try to keep an (arguably) good practice of only committing version bumps in a specific commit __with__ the proper `CHANGELOG` update.

Usually, I may decide to pack new changes in the new release, and then it makes no sense to bump it here : )

Reply:
1 - I'm AFK, so i'm using github web, i will revert in another commit, because i can't rebase right now.
2 -  Done.
3 - After this change i try install both 2 and 3 and works!